.github/workflows/storage-compatibility-check.yaml

```diff
@@ -52,48 +52,60 @@ jobs:
 
   integration-test-with-cosmos:
     name: Integration testing with the Consensus Commit transaction manager on Cosmos DB
-    runs-on: ubuntu-latest-l
+    runs-on: windows-latest
 
     steps:
       - name: Checkout the target repository
         uses: actions/checkout@v4
         with:
           ref: ${{ inputs.target-ref }}
 
-      - name: Prepare own schema.json
```

Comment:
Cosmos DB is no longer shared among tests for each branch (`master` and `3`), so we don't have to care about the namespace prefix.